src/messages.cpp

```diff
@@ -31,6 +31,12 @@ namespace quicr::messages {
                                 std::optional<Extensions>& extensions,
                                 std::optional<uint64_t>& current_tag)
     {
+
+        // get the count of extensions
+        if (!count && !ParseUintVField(buffer, count)) {
```

Comment:
Prefer not coercing integer type into bool
```suggestion
        if (count == 0 && !ParseUintVField(buffer, count)) {
```